shopfloor_reception_mobile/static/src/scenario/reception.js

```diff
@@ -346,14 +346,18 @@ const Reception = {
                 ],
             };
         },
-        picking_detail_options_for_select_move: function (move) {
+        picking_detail_options_for_select_move: function () {
             return {
                 show_title: true,
                 showActions: false,
                 list_item_options: {
                     loud_title: true,
                     title_action_field: {
-                        action_val_path: "name",
+                        action_val_path: function (record, field) {
+                            return record.barcode
```

Comment:
The record is move this doesn't contain a barcode barcode. It the product of the move should be checked instead. See https://github.com/OCA/wms/blob/867381a1a487a6e87a32179d81f233188a50d10a/shopfloor/actions/data.py#L243 https://github.com/OCA/wms/blob/867381a1a487a6e87a32179d81f233188a50d10a/shopfloor/actions/data.py#L304
```suggestion
                            return record.product.barcode
```

Reply:
Of course